### ISSUE TITLE
Can I create another sandbox in a different directory?

### ISSUE THREAD

**Issue by NotepadPlusUser** [sandboxie-plus/Sandboxie#2319]
### Is your feature request related to a problem or use case?

I have five sandboxes, all created within a standard Sandboxie directory 
D:\Sandbox\<name>\

What I would like to do is create another sandbox within the RAMDisk R: (I have 16MB of RAM, and at the moment I am using OSFMount to create the 1GB RAMDisk R: at each boot).

REASON: I would like to run my browsers mostly in the RAMDisk sandbox — that would avoid a lot of the churning and wearing down on the disks that normally goes on.

Is it possible to create another sandbox in a different directory from the other sandboxes?  

### Describe the solution you'd like

I want to keep the sandboxes in D:\Sandbox\<name>\ and create another sandbox in the RAMDisk R:

### Describe alternatives you've considered

I could place all my sandboxes within the RAMDisk, in a directory R:\Sandbox\<name>\ created automatically at each boot.  But that would often be inconvenient when downloading multiple files from the internet.

**Comment by RandomGOTI** [comment]
add `FileRootPath=R:\%SANDBOX%` to the temp sandbox you want using the ini tab (plus ui) or edit sandboxie.ini with a text editor of your choice

**Comment by isaak654** [comment]
The line provided above is the effective solution, it's just that no UI is planned for this functionality (see #1327).

**Comment by NotepadPlusUser** [comment]
Magic!  The standard Sandboxie directory appeared in R: drive as soon as I opened Vivaldi in the sandbox, and everything went as it should have.

Thanks you very much, RandomGOTI and Isaak654.  

Because I had some confusion working out what to do (not being a techie), I will list the steps very simply below — it probably needs editing, but I do think that careful instructions need to be written into the manual.
 - - - - - - - -

To add a sandbox called, say 'ExtraBox', to a location, say R:\MyFolder, different from the other sandboxes:
0.  First, for safety, save a backup copy of Sandboxie's INI file, which is normally C:\Windows\Sandboxie.ini 
1.  Open the Sandboxie UI --> Sandbox tab --> Create New Box, and type 'ExtraBox' into the 'Sandbox Name' field.
2.  Sandboxie UI --> Options tab --> Edit INI file, then 'OK' to the warning and 'Yes' to the UAC window.
3.  Scroll right down near the bottom to find the heading [ExtraBox].
4.  Just below this heading, add the single line: 
     FileRootPath=R:\MyFolder\%SANDBOX%
5.  Save and close the INI file, then configure and test the sandbox by
     Sandboxie UI --> Right-click on ExtraBox --> The usual steps.

**Comment by NotepadPlusUser** [comment]
I have now duplicated my 5 sandboxes with another 5 sandboxes created at each boot on R: drive, which is a 2GB Ramdrive cut from the 16GB of RAM using OSFMount.  I have also made the necessary adjustments to shortcuts. 

I find that I am using these RamDrive sandboxes now almost exclusively.  They are very convenient, because they are quick to delete, and secure deletion with overwriting is not needed because everything disappears on shutdown.  (Just don't forget to rescue any Downloads before shutdown.)

Thanks again, RandomGOTI and Isaak654, and of course DavidXanatos, for your help.
